src/renderer/util/index.js

```diff
@@ -168,3 +168,8 @@ export const animatedScrollTo = function (element, to, duration, callback) {
   }
   requestAnimationFrame(animateScroll)
 }
+
+export const getOptionsFromState = state => {
```

Comment:
maybe there is no need to put it in `util` file? because it will not used in other file other than `store/editor.js`.

So I think put it in `store/editor.js` is much better?

Reply:
Where inside `editor.js`? At the top of the file?

Reply:
yes, put it at the top of the file.